internal/pkg/util/fs/layout/layer/underlay/underlay_linux.go

```diff
@@ -143,14 +155,14 @@ func (u *Underlay) createLayer(rootFsPath string, system *mount.System) error {
 
 func (u *Underlay) duplicateDir(dir string, system *mount.System, existingPath string) error {
 	binds := 0
-	path := filepath.Clean(u.session.RootFsPath() + dir)
```

Comment:
When eliminating `Clean` here, is there any impact if `dir` is e.g. `../../../../` ?

Reply:
Because `filepath.Join` call `filepath.Clean` on returned result

Reply:
:+1: I was not aware of that, apologies.